llvm/lib/Target/AMDGPU/BUFInstructions.td

```diff
@@ -390,6 +392,7 @@ class MUBUF_Invalidate <string opName, SDPatternOperator node = null_frag> :
   let has_slc     = 0;
   let has_sccb    = 0;
   let sccb_value  = 0;
+  let tfe         = 0;
```

Comment:
Could maybe set it to 0 in the base class (BUF_Pseudo) instead of in three different subclasses?

Reply:
Yes. Done.